server/src/data_store.cpp

```diff
@@ -206,6 +222,7 @@ bool DataStore::RunQuery(const std::string& sql)
 
 int DataStore::DbCallback(void* not_used, int argc, char** argv, char** azColName)
 {
+    (void)not_used;
```

Comment:
Is this for future-proofing the interface?

Reply:
Sqlite3 requires this callback function with this signature for some of its operation. It only happens the variable is not used in this case. Actually, the name is a bad choice copied from examples.